_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+circuit. The fundamental building block that only made microchips possible is
+the **transistor**. Maybe you have heard about them before (processor marketing
+likes to include statements like "_Our new processor has XY billion
+transistors!_"). A transistor is nothing but a _switch_. However, unlike a
+_mechanical_ switch, a transistor's switching behavior can be controlled by some
+other _current_ (or voltage). This allows the transistor to be much faster and
+smaller.
+
+The first transistor was invented way back in 1947 (!) and could be held in one
+hand. Since then the sizes of transistors have been shrinking - and they still
+do today. Modern chips contain _billions_ of transistors. When you hear terms
+like "5nm CPU", that means nothing but the size of one transistor on that chip
+(actually that is not entirely true, but that's another story). So, transistors
+nowadays are only in the range of one _billionth_ of a meter in size! You might
+have heard about "Moore's Law" before. The "law" was named after Intel
+co-founder Gordon Moore. In 1965, he predicted that the number of transistors in
+chips would double about every two years. And in fact, this _exponential_ trend
+kept continuing for several decades, thanks to the advancements in transistor
+technology.
+
+<figure style="width:60%" class="align-center">
+  <img src="/assets/images/dd-00001-transistors-per-microprocessor.svg" alt="">
+  <!-- <figcaption class="text-center"></figcaption> -->
+</figure>
+
+But what makes transistors so special for chips? As we said earlier, a
+transistor behaves like a switch. You probably know that any switch (eg, a light
+switch) can be either "on" or "off", either "closed" or "open", either "1" or
+"0". (Maybe, in some reader's heads, some bells started ringing, but if not that
+is totally fine, too - that's why we created Bits&Volts :) This means a
+transistor can be in two _states_ (on-off, 1-0). And this is also precisely the
+reason why computers "think" in 1s and 0s, or _binary_. However, one transistor
+on its own cannot do much useful work for us. We have to connect multiple such
+transistors to form more complex circuits, that then can perform the tasks we
+designed them for. But how do these circuits look like? These circuits are
+composed of so-called **logic gates**, which we will cover next.
+
+## Logic Gates
+
+<figure style="width:30%" class="align-right">
+  <img src="/assets/images/dd-00001-logic-gates.svg" alt="">
+  <figcaption class="text-center">Logic Gates.</figcaption>
+</figure>
+
+A logic gate is made up of a couple of transistors and has inputs and outputs.
+Each input can be either a "1" or "0". A logic gate now produces a value at its
+outputs (for each output, either 1 or 0) based on what values are present at its
+inputs. There are different kind of logic gates. The most basic ones are AND,
+OR, and NOT gates. For instance, an AND gate has two inputs and one output. It
```

Comment:
In your image, you have a XOR instead of a NOT. So, do you want to use XOR in your text as well?

Reply:
I think I will stick with NOT, but then add a NOT gate to the picture as well.

Reply:
fca8941

Added the NOT picture. But also mentioned XOR in the text now.